src/common/lib/api.ts

```diff
@@ -33,64 +27,23 @@ export interface AccountInfoRes {
   name: string;
 }
 
-interface StatusRes {
+export interface StatusRes {
```

Comment:
@rolznz I export the interfaces to be able to use them in AccountContext. Would it make sense to move them to "src/types.ts"?

Reply:
That makes sense to me but let's leave them here for now to reduce the PR changes